src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -163,6 +166,21 @@ public TaskList getTaskList() {
         return taskList;
     }
 
+    /**
+     * Returns an unmodifiable view of the list of {@code Person} backed by the internal list of
+     * {@code versionedAddressBook}
+     */
```

Comment:
Should this comment should be regarding tasks instead of person?

Reply:
Fixed! Thank you.